src/pkg/packager/creator/skeleton.go

```diff
@@ -0,0 +1,313 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+
+// Package creator contains functions for creating Zarf packages.
+package creator
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/defenseunicorns/zarf/src/config/lang"
+	"github.com/defenseunicorns/zarf/src/extensions/bigbang"
+	"github.com/defenseunicorns/zarf/src/internal/packager/helm"
+	"github.com/defenseunicorns/zarf/src/pkg/layout"
+	"github.com/defenseunicorns/zarf/src/pkg/message"
+	"github.com/defenseunicorns/zarf/src/pkg/utils"
+	"github.com/defenseunicorns/zarf/src/pkg/utils/helpers"
+	"github.com/defenseunicorns/zarf/src/types"
+	"github.com/mholt/archiver/v3"
+)
+
+var (
+	// veryify that SkeletonCreator implements Creator
+	_ Creator = (*SkeletonCreator)(nil)
+)
+
+// SkeletonCreator provides methods for creating skeleton Zarf packages.
+type SkeletonCreator struct {
+	createOpts  types.ZarfCreateOptions
+	publishOpts types.ZarfPublishOptions
+}
+
+// NewSkeletonCreator returns a new SkeletonCreator.
+func NewSkeletonCreator(createOpts types.ZarfCreateOptions, publishOpts types.ZarfPublishOptions) *SkeletonCreator {
+	return &SkeletonCreator{createOpts: createOpts, publishOpts: publishOpts}
```

Comment:
```suggestion
	return &SkeletonCreator{createOpts, publishOpts}
```

With Go, if you name a variable the same as the property, and the types are in the correct order, you can omit the property name on struct instantiation.